Packs/Pwned/Integrations/PwnedV2/PwnedV2.py

```diff
@@ -66,15 +65,18 @@ def http_request(method, url_suffix, params=None, data=None):
         wait_regex = re.search(r'\d+', res.json()['message'])
         if wait_regex:
             wait_amount = wait_regex.group()
-
-            if datetime.now() + timedelta(seconds=int(wait_amount)) > RETRIES_END_TIME:
-                return_error('Max retry time has exceeded.')
-
-            time.sleep(int(wait_amount))
+        else:
+            demisto.error('failed extracting wait time will use default (5). Res body: {}'.format(res.text))
+            wait_amount = 5
+        if datetime.now() + timedelta(seconds=int(wait_amount)) > RETRIES_END_TIME:
+            return_error('Max retry time has exceeded.')
+        time.sleep(int(wait_amount))
 
     if res.status_code == 404:
         return None
     if not res.status_code == 200:
+        if not res.status_code == 401:
+            demisto.error('Error from Pwned v2: {}'.format(res.text))
```

Comment:
Include also the status_code and reason. It will make life easier when searching the logs.